src/App.jsx

```diff
@@ -14,45 +16,106 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
-export const App = () => (
-  <div className="section content">
-    <div className="buttons">
-      <button
-        type="button"
-        className="button is-info is-light"
-      >
-        Sort alphabetically
-      </button>
-
-      <button
-        type="button"
-        className="button is-success is-light"
-      >
-        Sort by length
-      </button>
-
-      <button
-        type="button"
-        className="button is-warning is-light"
-      >
-        Reverse
-      </button>
-
-      <button
-        type="button"
-        className="button is-danger is-light"
-      >
-        Reset
-      </button>
-    </div>
+export const App = () => {
+  const [visibileGoods, setVisibileGoods] = useState(goodsFromServer);
+
+  const [isAlphActive, setIsAlphActive] = useState(false);
+  const [isLengthActive, setIsLengthActive] = useState(false);
+  const [isReverseActive, setIsReverseActive] = useState(false);
+  const [resetButtonVisible, setResetButtonVisible] = useState(false);
+
+  const sortByAlph = () => {
+    setVisibileGoods(
+      [...visibileGoods].sort((goods1, goods2) => goods1.localeCompare(goods2)),
+    );
```

Comment:
The 'localeCompare' method is not case-insensitive. Please use toLowerCase() or toUpperCase() before comparing.